lib/Path/Tiny.pm

```diff
@@ -1037,8 +1039,20 @@ sub filehandle {
       unless defined $binmode;
     $binmode = "" unless defined $binmode;
 
+    # allow flock to be explicitly turned off via env PERL_PATH_TINY_NO_FLOCK=1
+    # and provide one warning because you don't want to turn this on by accident
+    my $no_flock = 0;
+    if ( defined $ENV{PERL_PATH_TINY_NO_FLOCK} && $ENV{PERL_PATH_TINY_NO_FLOCK} ) {
```

Comment:
It seems much clearer to me to say

````perl
my $no_flock = defined $ENV{PERL_PATH_TINY_NO_FLOCK} && $ENV{PERL_PATH_TINY_NO_FLOCK};
````

rather than

````perl
my $no_flock = 0;
if (defined $ENV{PERL_PATH_TINY_NO_FLOCK} && $ENV{PERL_PATH_TINY_NO_FLOCK}) {
    # ... code which doesn’t affect $no_flock ...
    $no_flock = 1;
}
````

Then you can also collapse the two conditionals which also makes things clearer still.

Reply:
In fact the `defined` check is superfluous since you just use the truthiness of the variable anyway, which is false for undefineds anyway. So this should really be

````perl
my $no_flock = !!$ENV{PERL_PATH_TINY_NO_FLOCK};
````